recipe/custom_toolchain/BUILD

```diff
@@ -7,12 +7,15 @@ filegroup(
 
 filegroup(
     name = "cc_wrapper",
-    srcs = ["cc_wrapper.sh"],
```

Comment:
with this bazel toolchain stuff, it looks like this is the updated (v0.2.0) toolchain files, plus modifications that were done specifically for tensorflow. i.e. you diffed the custom_toolchain folder in conda-forge's tensorflow-feedstock with the version of bazel-feedstock it was copied from; then updated the custom_toolchain here to our latest; then re-applied these diffs? Visually, the only thing I caught extra here is the patchelf bit..?

Reply:
Yes, this matches what I remember.

Reply:
ok, cool. It could be nice to document this approach for whoever does the next tensorflow update. Even better, some way of using the normal toolchain files and adding in the diff somehow (maybe using sed in the build.sh script?) would be good

Reply:
The way forward should be to update the normal toolchain and use it.

Reply:
would that mean patchelf needs to be included in the deps for any recipes that use it? If not, is there scope for updating the normal toolchain and using it before merging this PR?